app/views/journals/index.html.slim

```diff
@@ -16,11 +16,11 @@ h1= t_title(:index, Journal)
               td= l(record.created_at)
               td= t("category.#{record.category}")
               td= record.subject
-              td= record.body.truncate(40) if record.body
+              td= record.body&.truncate(40)
               td= link_to record.user.full_name, record.user.profile
-              td= link_to t_action(:show), client_journal_path(@journaled, record)
-              td= link_to t_action(:edit), edit_client_journal_path(@journaled, record)
-              td= link_to t_action(:destroy), client_journal_path(@journaled, record), confirm_deleting(record)
+              td= link_to t_action(:show), params.to_unsafe_hash.merge(action: :show, id: record.id)
+              td= link_to t_action(:edit), params.to_unsafe_hash.merge(action: :edit, id: record.id)
+              td= link_to t_action(:destroy), params.to_unsafe_hash.merge(action: :destroy, id: record.id), confirm_deleting(record)
```

Comment:
Can you try using `url_for(action: :show, id: record.id)` etc. here?

Or actually, I think you can even do `link_to ..., action: :show, id: record.id` without an explicit `url_for` call, though you need to wrap the params in braces for destroy since you're also passing HTML options there.

Reply:
The think is, that the partial is used from two different controllers (clients and volunteers), so the partial needs to make two different urls.
I find the generally phantastic Rails documentation quite a bit improvable if it comes to the link_to helper. There is a quadzillion different ways to use the thing.